### ISSUE TITLE
Specify the number of times the response to be mocked

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#261]
**Expected behavior**
The user should be able to specify the number of times the response to be mocked

**Current behavior**
It always gives the mocked response

**Suggestion**
`intercept(url, {numberOfTimes:2,body: responseBody })`
By default it is the current behavior where it will return the responseBody always

**Version**
```
commit - 97d15d97306e595b3a7275df2077bf0a6a64856b
```

**Comment by sriv** [comment]
Also consider allowing different response for each call. Ex. first call to `url` could be a success with a 201-Created. Subsequent could be a 200-OK, or 403-Unauthorized etc. The body could also be different.

This could either be such that it is a `playback` of multiple intercepts, or be defined explicitly via callback.

**Comment by SudhiHathwar** [comment]
+1  Currently blocked by this.

**Comment by rparthas** [comment]
Add a simple count to specify the number of requests to intercept. https://github.com/getgauge/taiko/pull/864
I will add another pull request if need to specify the intercept responses for various calls

**Comment by Apoorva-GA** [comment]
Available in commit : 9b9eb8c05c5a31cb923e555812dc35d9cbbd9d16

**Comment by Debashis9012** [comment]
Observation:
This above requirement can be achieved by following the below syntax:
```
block URL twice:
await intercept(url, undefined, 2)
mockResponse only 3 times :
await intercept(url, {mockObject}, 3)
```
Based on the above observation this issue has been verified and found fixed. 
Tested version:
`Version: 1.0.3 (Chromium: 76.0.3803.0) 9b9eb8c05c5a31cb923e555812dc35d9cbbd9d16`
